catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -19,123 +19,68 @@
 import java.util.Date;
 import java.util.function.Function;
 
+import org.codice.alliance.catalog.core.api.types.Security;
 import org.codice.imaging.nitf.core.common.DateTime;
 import org.codice.imaging.nitf.core.header.NitfHeader;
 
 import ddf.catalog.data.AttributeDescriptor;
 import ddf.catalog.data.AttributeType;
 import ddf.catalog.data.impl.AttributeDescriptorImpl;
 import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.types.Contact;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.data.types.Media;
 
 /**
  * NitfAttributes to represent the properties of a NitfFileHeader.
  */
 public enum NitfHeaderAttribute implements NitfAttribute<NitfHeader> {
-    FILE_PROFILE_NAME("fileProfileName",
+    FILE_PROFILE_NAME(Media.FORMAT,
             "FHDR",
             header -> header.getFileType()
                     .name()),
-    FILE_VERSION("fileVersion",
+    FILE_VERSION(Media.FORMAT_VERSION,
             "FVER",
             header -> header.getFileType()
                     .name()),
-    COMPLEXITY_LEVEL("complexityLevel",
-            "CLEVEL",
-            NitfHeader::getComplexityLevel,
-            BasicTypes.INTEGER_TYPE),
-    STANDARD_TYPE("standardType", "STYPE", NitfHeader::getStandardType),
-    ORIGINATING_STATION_ID("originatingStationId", "OSTAID", NitfHeader::getOriginatingStationId),
+    ORIGINATING_STATION_ID(Core.SOURCE_ID,
+            "OSTAID",
+            NitfHeader::getOriginatingStationId),
     FILE_DATE_AND_TIME("fileDateAndTime",
```

Comment:
For the fields that are not mapped directly here, the enum is still used in the code to `getAccessorFunction`. The `FILE_DATE_AND_TIME` accessor function in particular uses the `convertNitfDate` helper function in this class, so I kept it in the list.